test/cmd.js

```diff
@@ -115,7 +115,7 @@ describe('express(1)', function () {
       it('should create basic app', function (done) {
         run(ctx.dir, [], function (err, output) {
           if (err) return done(err)
-          assert.equal(parseCreatedFiles(output, ctx.dir).length, 17)
```

Comment:
The extra file is added to more elegantly support the transpiling. I recall this issue was raised earlier, although I can't find the exact comment, but the suggested solution was to break out the extra file. 

I think it may make more sense to explain by just showing the result of running `express --es2015` 
<img width="929" alt="example" src="https://cloud.githubusercontent.com/assets/2670813/21458327/3e855e7c-c8fb-11e6-9f9b-65ce18bc614e.png">

I vaguely remember the original concern was that the resulting generated `www` file didn't use the es2015 syntax itself, due to how `babel-core/register` [works](https://babeljs.io/docs/usage/babel-register/). So, the idea was to keep `bin/www` as the entrypoint to the application, but breaking the actual server code into into `server.js`, which is located in the root dir of the generated app.